packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -309,6 +309,12 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     message.attached === 'top' || message.attached === false
       ? chatMessageStyles.bodyWithAvatar
       : chatMessageStyles.bodyWithoutAvatar,
+    // handle rtl margins for message without avatar
+    theme.rtl && !(message.attached === 'top' || message.attached === false)
+      ? shouldOverlapAvatarAndMessage
+        ? chatMessageStyles.avatarOverlapRTL
```

Comment:
Can we keep the naming consistent between the one above and this one? I'm good with either name.

So maybe something like:

chatMessageStyles.bodyWithAvatarRTL
chatMessageStyles.bodyWithoutAvatarRTL

Reply:
Updated